src/genie/libs/parser/viptela/show_control_local_properties.py

```diff
@@ -0,0 +1,453 @@
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any
+from pprint import pprint
+import re
+
+# ===============================================
+# Schema for 'show control local-properties'
+# ===============================================
+
+class ShowControlLocalPropertiesSchema(MetaParser):
+
+    """ Schema for "show control local-properties" command """
+
+    schema = {
+        "personality": str,
+        "sp_organization_name": str,
+        "organization_name": str,
+        "root_ca_chain_status": str,
+        "certificate_status": str,
+        "certificate_validity": str,
+        "certificate_not_valid_before": str,
+        "certificate_not_valid_after": str,
+        "enterprise_cert_status": str,
+        "enterprise_cert_validity": str,
+        "enterprise_cert_not_valid_before": str,
+        "enterprise_cert_not_valid_after": str,
+        "dns_name": str,
+        "site_id": str,
+        "domain_id": str,
+        "protocol": str,
+        "tls_port": str,
+        "system_ip": str,
+        "chassis_num_unique_id": str,
+        "serial_num": str,
+        "enterprise_serial_num": str,
+        "token": str,
+        "keygen_interval": str,
+        "retry_interval": str,
+        "no_activity_exp_interval": str,
+        "dns_cache_ttl": str,
+        "port_hopped": str,
+        "time_since_last_port_hop": str,
+        "pairwise_keying": str,
+        "embargo_check": str,
+        "number_vbond_peers": str,
+        "number_active_wan_interfaces": str,
+        "connection_summary": {
```

Comment:
just suggest key name.
```suggestion
        "wan_interfaces": {
```